Dockerfile

```diff
@@ -1,57 +1,5 @@
-FROM ubuntu:14.04
+FROM ubuntu:16.04
```

Comment:
@LuisAlejandro I'd vote put new version in a separate PR so we can test if the new packages versions still works with Odoo (we had a lot of them when we moved from previous LTS)